emp/iam_policy.json

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
Lets have name of file to be `aws-policy.json` bcz UI is consuming same. https://raw.githubusercontent.com/platform9/support-locker/master/emp/aws-policy.json